internal/engine/image_build_and_deployer_test.go

```diff
@@ -261,6 +261,23 @@ func TestKINDPush(t *testing.T) {
 
 	assert.Equal(t, 1, f.docker.BuildCount)
 	assert.Equal(t, 1, f.kp.pushCount)
+	assert.Equal(t, 0, f.docker.PushCount)
+}
+
+func TestCustomBuildDisablePush(t *testing.T) {
+	f := newIBDFixture(t, k8s.EnvKIND)
+	defer f.TearDown()
+
+	manifest := NewSanchoCustomBuildManifestWithPushDisabled(f)
+	_, err := f.ibd.BuildAndDeploy(f.ctx, f.st, buildTargets(manifest), store.BuildStateSet{})
+	// the image doesn't exist, so the custom build fails
+	// TODO(dmiller): change this test to use the static tag parameter when that's added
+	assert.Error(t, err)
```

Comment:
can you add an assertion that this is the error you expect, and not some other error?

Reply:
Yes, but I probably should have made this more clear: this test doesn't do anything at the moment because the tag is generated by the build and deployer and I can't know what it is. Thus the TODO to change this test to use the static tag parameter that @maiamcc is going to add. So really this is more of a stub, and once that parameter is added I can inject the static image tag in to the fake docker client and the build and deployer won't return an error at all.